src/spec/AnimationSpec.ts

```diff
@@ -48,4 +50,28 @@ describe('An animation', () => {
     expect(animation.speed).toBe(20);
     expect(animation.sprites.length).toBe(0);
   });
+
+  it('should always pass "flipped" state to the current Sprite', () => {
+    const mockSprite = {
+      anchor: new ex.Vector(1, 1),
+      draw: () => void 0,
```

Comment:
Even though these are effectively throwaway values, `anchor` and `draw` were required to be defined in some way in order to prevent the tests from crashing when "drawing" a Sprite.